packages/viewer/src/lib/scenes/camera.ts

```diff
@@ -148,7 +148,7 @@ export abstract class Camera {
     );
     const cameraToCenter = Vector3.subtract(position, boundingBoxCenter);
 
-    return (
+    return -(
       Vector3.dot(viewVector, cameraToCenter) / Vector3.magnitude(viewVector)
     );
```

Comment:
Ended up flipping this value based on some testing. Basically what I found is that a camera defined as:
```typescript
{
  lookAt: { x: 0, y: 0, z: 0 },
  up: { x: 0, y: 1, z: 0 },
  position: { x: 0, y: 0, z: 5 },
}
```
looking at a bounding box centered at the origin, would return a distance of `-5` for a distance of 5 toward the `lookAt`. Moving the `lookAt` to be `{ x: 0, y: 0, z: 10 }` (effectively 180 turn), the distance was `5`, but away from the `lookAt`. There may be a better alternative than just flipping the value however, so I'm open to other ideas